test/controllers/courses_controller_test.rb

```diff
@@ -159,6 +176,27 @@ def test_update_should_hide_course
     assert_response :ok
   end
 
+  def test_update_should_unhide_course
+    course_name = 'fake course_name'
+    course_organization = "course_org"
+
+    stub_organization_membership_admin_in_org(course_organization, ENV["MACHINE_USER_NAME"])
+    stub_organization_is_an_org(course_organization)
+
+    assert_nil Course.find_by(name: course_name)
+    course = Course.create(name: course_name, course_organization: course_organization, hidden: false)
+
+    patch course_url(course), params: { course: { hidden: 0 } }
+
+    after = Course.find_by(name: course_name)
+    assert_not_nil after
+    refute after.hidden
+
+    get root_url
+    assert_response :ok
+    assert response.body.scan(course_name).count > 0
+  end
+
```

Comment:
Brian confirms that this is not necessarily related to this PR... it's a test that was commented out becuase it was broken and he fixed it.  

But, it's probably ok to leave it in this PR since it's helping to improve our test coverage.